packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## 43.16.0 - September 13, 2018
+
+### Fixed
+
+#### salesforcedx-vscode-apex-debugger, salesforcedx-vscode-apex-replay-debugger
+
+- Prevent Apex Debugger and Apex Replay Debugger from being activated in non sfdx projects ([PR #631](https://github.com/forcedotcom/salesforcedx-vscode/pull/631))
+
+#### salesforcedx-vscode-apex-replay-debugger
+
+- Allow breakpoints to be set and hit after heapdumps have been fetched ([PR #609](https://github.com/forcedotcom/salesforcedx-vscode/pull/609))
```

Comment:
Not sure if we need to include this one yet since this isn't out yet.

Reply:
Np, I'll remove it